keyboards/iris/rules.mk

```diff
@@ -1,36 +1,9 @@
-# MCU name
-MCU = atmega32u4
 
-# Processor frequency.
-#     This will define a symbol, F_CPU, in all source code files equal to the
-#     processor frequency in Hz. You can then use this symbol in your source code to
-#     calculate timings. Do NOT tack on a 'UL' at the end, this will be done
-#     automatically to create a 32-bit value in your source code.
-#
-#     This will be an integer division of F_USB below, as it is sourced by
-#     F_USB after it has run through any CPU prescalers. Note that this value
-#     does not *change* the processor frequency - it should merely be updated to
-#     reflect the processor speed set externally so that the code can use accurate
-#     software delays.
-F_CPU = 16000000
+SRC = keyboards/wilba_tech/wt_main.c
```

Comment:
Maybe this should only be added if dynamic keymaps is enabled?

Reply:
After playing around with moving around some of these lines into common_features.mk, I figure it's best to let @Wilba6582 to handle the refactoring around that, to allow for user/keymap level of disabling dynamic keymaps in their rules.mk.